chirho/dynamical/handlers/event_loop.py

```diff
@@ -28,6 +29,9 @@ class InterruptionEventLoop(Generic[T], pyro.poutine.messenger.Messenger):
     def _pyro_simulate(msg) -> None:
         dynamics, state, start_time, end_time = msg["args"]
 
+        if pyro.settings.get("validate_dynamics"):
+            check_dynamics(dynamics, state, start_time, end_time, **msg["kwargs"])
```

Comment:
@SamWitty I went ahead and added the validation flag to this PR instead of making a new one.

Reply:
@eb8680, while we're waiting for #380 to land, could you add this `check_dynamics` call to `simulate` as well?